kernel/kernel-websubclient-api/src/main/java/io/mosip/kernel/websub/api/client/SubscriberClientImpl.java

```diff
@@ -151,4 +165,47 @@ public SubscriptionChangeResponse unSubscribe(UnsubscriptionRequest unsubscripti
 		}
 	}
 
+
+
+
+	@Override
+	public FailedContentResponse getFailedContent(FailedContentRequest failedContentRequest) {
+		HttpHeaders headers = new HttpHeaders();
+		headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
+		headers.set(WebSubClientConstants.SUBSCRIBER_SIGNATURE_HEADER, getHmac256(failedContentRequest.getTopic()+failedContentRequest.getCallbackURL()+failedContentRequest.getTimestamp(),failedContentRequest.getSecret())); 
+
+		UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(failedContentRequest.getHubURL())
+		        .queryParam("topic", failedContentRequest.getTopic())
+		        .queryParam("callback", failedContentRequest.getCallbackURL())
```

Comment:
Since callback will be URL, can we base64URL encode and use that value rather?

Reply:
sure